.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -17,14 +15,15 @@ labels: bug
 
 ## How to reproduce
 
-1. 
-2. 
-3. 
+1.
+2.
+3.
 
 ## Environment
 
-- PostHog cloud or self-managed?
-- PostHog version/commit
+- PostHog Cloud or self-hosted?
+- PostHog plan (free, growth, enterprise etc.)?
```

Comment:
I'm thinking we can improve this. If the user is on cloud, we don't really care about version/commit (only on self-hosted). I'm also not sure about the benefit of asking for the user's plan (we could just ask or look it up if necessary) and it gives me the feeling that if I'm not on an enterprise plan, my bug report will not be read. But obviously feel free to challenge this.